IFC4x3/Properties/e/EndSmoothingLength_2EtJ89wAfDgPQfnCNoxkN6/Documentation.md

```diff
@@ -1 +1 @@
-Length for the circular transition at the end of the cant segment, measured from the start of the circular transition to the end of the segment.
+Length for the circular transition change of curvature placed at the end of the cant segment, measured from the start of the circular transition change of curvature to the end of the cant segment.
```

Comment:
Not sure I understand this fix. What do you think of the proposed amend?
```suggestion
Length for the circular transition change of curvature at the end of the cant segment, measured from the start of the circular transition change of curvature to the end of the cant segment.
```

Reply:
OK for me, thanks!